File: Pascal/Operaciones.pas
program Operaciones;
uses crt;
var
  a, b, i, n: integer;
  suma, resta, multiplicacion: integer;
  division: real;
  sumatoria: integer; 
begin
  clrscr;
  writeln('Hola Mundo');
  write('Ingrese el primer numero : ');
  readln(a);
  write('Ingrese un segundo numero: ');
  readln(b);
  
  suma := a + b;
  resta := a - b;
  multiplicacion := a * b;
  division := a / real(b);
  writeln('El resultado de la suma es =', suma);
  writeln('El resultado de la resta es =', resta);
  writeln('El resultado de la multiplicacion es =', multiplicacion);
  writeln('El resultado de la divicion es =', division:4:2);
  
  sumatoria := 0;
  n := 20;
  
  for i := 1 to n do begin
    sumatoria := sumatoria + i;
  end;
  writeln('Sumatoria = ', sumatoria);
  readkey;
end.

File: PHP/Operaciones.php
<?php
  // Operaciones Basicas en PHP
  echo "Hola Mundo!"; // Imprime el texto Hola Mundo! 
  $numero = 10;       // Declaramos una variable llamada numero
  // y le asignamos un valor de 10 a la misma.
  echo "<br>";      // Salto de línea para separar los resultados
  echo "El número es: ", $numero, ".<br>";    // Concaten
  echo "La suma de 5 + 3 es: ", 5+3,
  ".<br> La resta de 7 - 2 es: ", 7-2
  ."<br> La multiplicación de 4 * 2 es: ",
  4*2,".<br>";   
  $division = 10 / 2; 
  echo "La división =", $division;  // Realizamos una operacion
  echo "<br>";
  // de división entre dos números
  if ($division == 5) {        // Comprobamos si resultado es un decimal
    echo "La división es exacta.";
    } else {                   
      echo "La división no es exacta.";
      }
  ?>